src/RMP/Translate/Translate.php

```diff
@@ -90,6 +90,11 @@ public function translate($key, $substitutions = array(), $pluralisation = null,
         }
         $result = $this->_translate($key, $substitutions, $pluralisation, $domain, $this->locale);
 
+        // test if the locale is the 2 letter variant which is oftent taken from ckps_language
+        if ($result === '' && strlen($this->locale) == 2) {
+            $result = $this->_translate($key, $substitutions, $pluralisation, $domain, $this->locale . '_GB');
+        }
```

Comment:
No, but the brits have claimed stakes on many Spanish counties. There are probably quite a few `es_GB` regions :)

Yea I considered this scenario and figured it would be empty upon checking but whitelisting makes more sense.